kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/KafkaUtils.java

```diff
@@ -81,7 +72,42 @@ private static String consumeMessages(String deployNamespace, String topicName,
      * @return the log of the pod
      */
     public static String consumeMessageWithTestClients(String deployNamespace, String topicName, String bootstrap, int numOfMessages, Duration timeout) {
-        return consumeMessages(deployNamespace, topicName, bootstrap, numOfMessages, " - " + (numOfMessages - 1), timeout);
+        String name = Constants.KAFKA_CONSUMER_CLIENT_LABEL;
+        Job testClientJob = TestClientsJobTemplates.defaultTestClientConsumerJob(name, bootstrap, topicName, numOfMessages).build();
+        return consumeMessages(topicName, name, deployNamespace, testClientJob, " - " + (numOfMessages - 1), timeout);
+    }
+
+    /**
+     * Consume messages with kcat.
+     *
+     * @param deployNamespace the deploy namespace
+     * @param topicName the topic name
+     * @param bootstrap the bootstrap
+     * @param messageToCheck the message to check
+     * @param timeout the timeout
+     * @return the string
+     */
+    public static String consumeMessagesWithKcat(String deployNamespace, String topicName, String bootstrap, String messageToCheck, Duration timeout) {
```

Comment:
wondering if it would be worth shuffling things around so instead of a growing methods on the KafkaUtils we access these bits of work through something like:
```
KafkaClients.KCAT.produce("namespace", "topic", "bootstrap", "message", 10)
KafkaClients.KCAT.consume("namespace", "topic", "message-9", Duration.ofSeconds(20))
```
with a common interface for the `Client` types